src/methods_transcript_assignment/pciSeq_transcript_assignment/script.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+import dask
+import spatialdata as sd
+import txsim as tx
+import anndata as ad
+import os
+import shutil
+
+## VIASH START
+# Note: this section is auto-generated by viash at runtime. To edit it, make changes
+# in config.vsh.yaml and then run `viash config inject config.vsh.yaml`.
+par = {
+  'input_ist': 'resources_test/task_ist_preprocessing/mouse_brain_combined/raw_ist.zarr',
+  'input_segmentation': 'resources_test/task_ist_preprocessing/mouse_brain_combined/segmentation.zarr',
+  'transcripts_key': 'transcripts',
+  'coordinate_system': 'global',
+  'output': '../pciSeq_assigned_transcripts.zarr',
+
+  'input_scrnaseq': '../brain_scrnaseq_subsample_WMB-10X.h5ad',
```

Comment:
Did you test with `task_ist_preprocessing/resources_test/task_ist_preprocessing/mouse_brain_combined/scrnaseq_reference.h5ad` ? 
good to stick with the given resources in the VIASH START END section